components/dropdown/dropdown-content-styles.js

```diff
@@ -0,0 +1,148 @@
+import { css } from 'lit-element/lit-element.js';

```

Comment:
Is this needed by more than 1 thing? If not, we should pull it into `dropdown-content.js`. Otherwise when we finally get unbundled builds working this will be an unnecessary extra request.

Reply:
At the moment, no. But it will be pulled in to [`dropdown-menu`](https://github.com/BrightspaceUI/dropdown/blob/master/d2l-dropdown-menu.js#L21) and [`dropdown-tabs`](https://github.com/BrightspaceUI/dropdown/blob/master/d2l-dropdown-tabs.js#L8) when they get added.

Given how similar they are, it could likely be converted to a component rather than a mixin but for the initial iteration I went keeping it as close to the original as possible to make it easier to review and reduce the risk of accidentally breaking something.

Reply:
Cool makes sense. If it's needed, it's needed!